### PR TITLE
add utiliy func scoreToReputation to common

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

File: Scripts/script-CommonServer.yml [hunk 1]
```diff
@@ -1103,6 +1103,21 @@ script: |-
       email: 'Account.Email(val.Address && val.Address == obj.Address)'
   };
 
+
+  // Converts score (in number format) to human readable reputation format
+  function scoreToReputation(score) {
```
Comment (dorsha) [inline Scripts/script-CommonServer.yml]: use `===` in all places.
And don't we have constants for these numbers?

File: Scripts/script-CommonServer.yml [hunk 2]
```diff
@@ -1103,6 +1103,21 @@ script: |-
...
+      }
+      if (score == 2){
+          return 'Suspicious';
+      }
+      if (score == 1){
+          return 'Good';
+      }
+      return 'None';
+  };
+
```
Comment (dorsha) [inline Scripts/script-CommonServer.yml]: I would also remove this and use your new func:

image

Comment (meirwah) [reply Scripts/script-CommonServer.yml]: don't understand the remark here.. what to remove?

Comment (dorsha) [reply Scripts/script-CommonServer.yml]: The entire function since you have it in common now

Comment (meirwah) [reply Scripts/script-CommonServer.yml]: that script is deprecated, only for version <=3.1, so can't use that func there

**Pull request merged**